types/react-beautiful-dnd/v12/index.d.ts

```diff
@@ -600,9 +597,9 @@ export interface DroppableProps {
     isCombineEnabled?: boolean | undefined;
     direction?: Direction | undefined;
     ignoreContainerClipping?: boolean | undefined;
-    renderClone?: DraggableChildrenFn | undefined;
-    getContainerForClone?: (() => React.ReactElement<HTMLElement>) | undefined;
-    children(provided: DroppableProvided, snapshot: DroppableStateSnapshot): React.ReactElement<HTMLElement>;
+    renderClone?: DraggableChildrenFn | null | undefined;
+    getContainerForClone?: (() => HTMLElement) | undefined;
+    children(provided: DroppableProvided, snapshot: DroppableStateSnapshot): React.ReactElement<HTMLElement> | null;
```

Comment:
Should we use `React.ReactNode` for `children` return types?

Reply:
Yes 😄